mongo_connector/__init__.py

```diff
@@ -11,7 +11,23 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+import logging
+
 from pkgutil import extend_path
 __path__ = extend_path(__path__, __name__)
 
-__version__ = '2.5.0.dev0'
+version_info = (2, 5, 0, 'dev0')
+version = '.'.join(str(v) for v in version_info)
+__version__ = version
```

Comment:
So I just realized there's a larger problem here which is that `mongo-connector` and the docManagers all share the same `mongo-connector` module.  This means that if you do:
```
pip install mongo-connector
pip install elastic2-doc-manager
```
 the `__init__py` files of `elastic2-doc-manager` will clobber the existing ones (`mongo_connector/__init__.py` and `mongo_connector/doc_managers/__init__.py`). This means we can't rely on anything inside these files. In my opinion, this should be changed so that third party doc managers live in separate modules. Perhaps I should add versions in another file until that is settled.